_data/simple-icons.json

```diff
@@ -12426,7 +12426,13 @@
 	{
 		"title": "Plus Codes",
 		"hex": "4285F4",
-		"source": "https://maps.google.com/pluscodes/"
+		"source": "https://maps.google.com/pluscodes",
+		"guidelines": "https://maps.google.com/pluscodes/brand-guidelines",
+		"aliases": {
+			"aka": [
+				"Open Location Code"
+			]
+		}
```

Comment:
See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#also-known-as. Can we just remove it?